tools/sloop_shad.c

```diff
@@ -0,0 +1,13 @@
+#include  <ansi.h>
+inherit AnsiTellCode;
+
+void sh_init(object ob )
+{
+    shadow( ob, 1 );
+    return;
+}
+
+string short()
+{
+    return( query_shadowing(THISO)->short() )+ BOLD_GREEN" [slooped]" NORM;
```

Comment:
I believe query_shadowing() returns the next shadow in the stack, not necessarily the host. In other words, if you shadow someone that's already being shadowed by something else, this might not do what you're expecting it to. Not 100% on this one would have to test it out to see what happens. It might actually do exactly what you want it to depending on the circumstances. We can talk about this more when we talk shadows.